src/confdb/data/PrimaryDataset.java

```diff
@@ -776,6 +786,31 @@ public boolean addDataset(PrimaryDataset dataset){
             }
         }
 
+        public Long getPathPrescale(String pathName){
+            Long prescale = Long.valueOf(-1);
+            for(String pFilterPar : this.pathFilterParam.values()){
+                String[] pFilterParSplit = pFilterPar.split(" / ");
+                String pName = pFilterParSplit[0];
+                if(pathName.equals(pName)){
+                    Long pPrescale = Long.valueOf(1);
+                    if(pFilterParSplit.length > 1){
+                        pPrescale = Long.valueOf(pFilterParSplit[1]).longValue();
+                    }
+                    if(prescale >= 0){
+                      System.out.println("PathFilter::getPathPrescale(\""+pathName+"\") -- ERROR: Found duplicate Path in PrimaryDataset");
```

Comment:
hmm I wonder if this is caught another way. That would be....bad and the whole thing is about to come crashing down. Note a problem with this PR of course. Just thinking outloud